docker-compose.yml

```diff
@@ -33,7 +35,7 @@ services:
     depends_on:
       - minio
   clamav-rest:
-    image: ajilaag/clamav-rest
+    image: kcirtapfromspace/clamav-rest
```

Comment:
I know this is yours, but I see that no one else is maintaining a cross platform version.
I think we need to reconfigure the clamav settings. As the import system is attempting to queue all the files its placing on s3 to be scanned. But they are ~75MB each.

Reply:
This is just to build arm64 on m1 machines. open pr to clam for the to build arm https://github.com/ajilach/clamav-rest/pull/37